t/t2407-sdbus.t

```diff
@@ -13,6 +13,10 @@ if ! systemctl --user show --property Version; then
 	skip_all="user systemd is not running"
 	test_done
 fi
+if ! busctl --user status >/dev/null; then
```

Comment:
commit message typo - to "skip the tests" (and not sure that has to be in double quotes?)

Reply:
a complete aside comment, should these fixes be copied/updated in `t2405-job-exec-sdexec.t` and `t2410-exec-systemd.t` too?  Possibly elsewhere too?

In those tests it checks

```
#  Don't run if systemd environment variables not setup
if test -z "$DBUS_SESSION_BUS_ADDRESS" \
     -o -z "$XDG_RUNTIME_DIR"; then
    skip_all='DBUS_SESSION_BUS_ADDRESS and/or XDG_RUNTIME_DIR not set'
    test_done
fi
```

which is probably why its not a problem, assuming the DBUS env is set correctly.

But perhaps it would be good to make the check consistent across all tests.  Could be a new issue and we can follow up later.

Reply:
I'm not sure at this point what we're going to end up keeping after we finally get `job-exec` working with the new approach, so I'd prefer we table that one for now.  Helper function seems like overkill to me right now.